From: Alexander Mikhaylenko <alexm@gnome.org>
Date: Fri, 9 Sep 2022 10:25:15 +0400
Subject: firmware-security: Don't use .view on rows

I'm not sure what the intention was here, but it didn't work. It made rows
white in light variant (same as other rows), but even darker than they were
before in dark.

Origin: upstream, 43.1, commit:53fd0d69e20ece846fabc539b902e36cdd1bbb64
---
 panels/firmware-security/cc-firmware-security-dialog.c | 1 -
 panels/firmware-security/cc-firmware-security-panel.c  | 1 -
 2 files changed, 2 deletions(-)

diff --git a/panels/firmware-security/cc-firmware-security-dialog.c b/panels/firmware-security/cc-firmware-security-dialog.c
index e5113db..2109aa5 100644
--- a/panels/firmware-security/cc-firmware-security-dialog.c
+++ b/panels/firmware-security/cc-firmware-security-dialog.c
@@ -270,7 +270,6 @@ hsi_create_pg_row (const gchar *icon_name,
     {
       GtkWidget *subrow = adw_action_row_new ();
       g_autofree gchar *str = fu_security_attr_get_description_for_dialog (attr);
-      gtk_widget_add_css_class (subrow, "view");
       adw_action_row_set_subtitle (ADW_ACTION_ROW (subrow), str);
       adw_expander_row_add_row (ADW_EXPANDER_ROW (row), subrow);
       gtk_widget_add_css_class (subrow, "security-description-row");
diff --git a/panels/firmware-security/cc-firmware-security-panel.c b/panels/firmware-security/cc-firmware-security-panel.c
index bf0ba70..d48e7ab 100644
--- a/panels/firmware-security/cc-firmware-security-panel.c
+++ b/panels/firmware-security/cc-firmware-security-panel.c
@@ -202,7 +202,6 @@ parse_event_variant_iter (CcfirmwareSecurityPanel *self,
     {
       GtkWidget *subrow = adw_action_row_new ();
       g_autofree gchar *str = fu_security_attr_get_description_for_eventlog (attr);
-      gtk_widget_add_css_class (subrow, "view");
       adw_action_row_set_subtitle (ADW_ACTION_ROW (subrow), str);
       adw_expander_row_add_row (ADW_EXPANDER_ROW (row), subrow);
     }
